package.json

```diff
@@ -156,6 +156,6 @@
     ]
   },
   "engines": {
-    "node": ">=14.17.5"
-  }
+      "node" : ">=16.0.0"
```

Comment:
We are gonna use 16 or 18?

Reply:
18 - this is given by the `.nvmrc`; what `engines` specifies is the version of node that our app will work with. 

i specified 16 because it'll work with 16+ (technically 14 is acceptable but we want to outlaw it)

Reply:
makes sense!